README.md

```diff
@@ -11,8 +5,82 @@
 1. Define SQL and understand what it is used for. 
 2. Install SQL on your computer
 
-## What is SQL
+## What is SQL?
+
+SQL (Structured Query Language) is a language for managing data in a database. Unlike some other sorts of programming languages, it's only used for one thing: talking to databases. Thus, you might hear it referred to as a "special purpose" programming language, which is really not all that terribly important to understand beyond knowing that you won't be using SQL to write the next big web app, but you might write some to interact with the database that powers it.
+
+Even though SQL really only has one purpose, it is used by many different database systems you might be familiar with (don't worry if you're not) such as MySQL, PostgreSQL, or the system we'll being using in this course: SQLite. All of these are relational database management systems. 
 
 ## Installation
 
-## Troubleshooting
\ No newline at end of file
+### Macs Make It Easy
+
+If you are on OSX version 10.4 or greater, you probably already have SQLite installed. Find out by opening up the terminal and pasting in:
+
+```bash
+which sqlite3
+```
+
+if you get back
+
+`/usr/bin/sqlite3`
+
+Then you have a working version of sqlite3 already installed on your system. Thanks Apple! Skip ahead to
```

Comment:
Maybe "Skip ahead to the next lesson" or "mark this lesson as complete and move on"?